eks-anywhere-common/Addons/Core/Daily-S3-Exporter/manifestCronJob.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: batch/v1beta1
```

Comment:
Is this starting 1.24? In other words, does it mean we won't support any environment below 1.24?

Reply:
This is starting `1.25` only. Still older version swill it work. `batch/v1` worked for me with 1.21, 1.22, 1.23 eksa